config/config.json

```diff
@@ -2,21 +2,21 @@
   "development": {
     "username": "Jessica",
     "password": null,
-    "database": "blog-app",
+    "database": "bookclub",
```

Comment:
get this from process.env like this: https://github.com/izelnakri/nycda-blog-application/commit/e929a158f3f3f3d20f63b853ad0af91986f9772f